x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/constants.ts

```diff
@@ -173,6 +187,13 @@ export const CONFIG_CUSTOM_BUTTON = i18n.translate(
   }
 );
 
+export const CONFIG_CUSTOM_LINK_TEXT = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSource.configCustom.docs.link.text',
```

Comment:
As per the i18n guidelines, can we change this to:
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.contentSource.configCustom.docs.linkText',
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697